app/views/layouts/_attachment_object.html.haml

```diff
@@ -0,0 +1,9 @@
+- if attachment
```

Comment:
can this be done with a helper rather than a layout, just to keep it all in ruby since the rest is?